plugins/destination/postgresql/client/client.go

```diff
@@ -117,20 +113,11 @@ func New(ctx context.Context, logger zerolog.Logger, spec specs.Destination) (pl
 	return c, nil
 }
 
-func (c *Client) Metrics() plugins.DestinationMetrics {
-	return c.metrics
-}
-
 func (c *Client) Close(ctx context.Context) error {
 	var err error
 	if c.conn == nil {
 		return fmt.Errorf("client already closed or not initialized")
 	}
-	if c.batch.Len() > 0 {
-		br := c.conn.SendBatch(ctx, c.batch)
-		err = br.Close()
-		c.batch = &pgx.Batch{}
-	}
```

Comment:
Was this moved somewhere else?

Reply:
yeah. batches are created on write as we don't want to reuse them between streams. and they are closed/send at the end of the stream rather then on close